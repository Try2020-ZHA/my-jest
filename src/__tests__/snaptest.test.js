import renderer from 'react-test-renderer';
import Contest from '../component/contest';
import React from 'react';

describe('the contest is rendered correctly',()=>{
    test('render it',()=>{
        const attr={id:1,text:'this is a test',done:true};
        const result=renderer.create(<Contest attr={attr}></Contest>);
        expect(result.toJSON()).toMatchSnapshot();
    })
})