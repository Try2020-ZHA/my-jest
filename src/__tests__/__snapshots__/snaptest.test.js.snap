// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the contest is rendered correctly render it 1`] = `
<div>
  <p
    id="text"
  >
    this is a test
  </p>
  <p>
    next paragraph
  </p>
  <button
    className="trueDone"
    onClick={[Function]}
  >
    button
  </button>
  <button
    className="fakeDone"
    onClick={[Function]}
  >
    button
  </button>
</div>
`;
